db/migrate/20161108111455_create_shipments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateShipments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :shipments do |t|
+    	t.integer :quantity
+    	t.integer :broker_id
+    	t.integer :forward_freight_id
```

Comment:
I already did this in another migration file, but whenever you have foreign keys, make sure to index them :)